website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -154,3 +154,7 @@ Workspaces may be configured to share their state with other workspaces within t
 ### Treat Archivist URLs as Secrets
 
 Terraform Cloud uses a blob storage service called Archivist for storing various pieces of customer data. Archivist URLs (which have the origin <https://archivist.terraform.io>) are returned by various Terraform Cloud APIs, such as the [state versions API](/terraform/cloud-docs/api-docs/state-versions#fetch-the-current-state-version-for-a-workspace). Unlike the rest of Terraform Cloud’s API, Archivist’s API does not require the user to submit a Bearer Token with each request. Instead, Archivist URLs contain a signed short term authorization token that is used to perform authorization checks. For this reason, Archivist URLs must be treated as secrets and should not be logged or shared.
+
+### Consider Using Dynamic Provider Credentials
```

Comment:
Could we shorten this to "Using dynamic credentials"? I think it reads a bit more clearly if folks aren't acquainted with "Dynamic Provider Credentials" yet, they still get from content what a dynamic credential is. 
```suggestion
### Using dynamic credentials
```

Reply:
Dang, I just realized I deleted on my own comment by accident! Anyways, **thank you** for these very thoughtful questions! 

1. [Our style guide states to use sentence case for all headers](https://github.com/hashicorp/engineering-docs/blob/main/writing/style-guide.md#titles-and-headers), but that update _might_ have happened after this article was originally written. It's not a huge priority for older articles, but we should use sentence case from now on!  

(_My bad alert: When I first joined I didn't read this part of the style guide closely enough, so I made some suggestions to use title case in the past, which is totally my bad!_ 😅 ) 

If have the bandwidth and wouldn't mind, it'd be amazing if you updated the headers for the rest of the article too, but no problem if you'd rather not! 

2. You are totally correct! We should use the imperative in headers, so correcting my suggestion below:
```suggestion
### Use dynamic credentials
```

Reply:
Updated.  Want to take another look?